packages/superset-ui-color/src/CategoricalColorScale.ts

```diff
@@ -78,4 +79,85 @@ export default class CategoricalColorScale extends ExtensibleFunction {
       ...this.parentForcedColors,
```

Comment:
It is intentional to allow color map being overridden by parent?

Reply:
Yes, a scale can be in a namespace. We allow override at namespace level. so that should override scale-level setting.